az-digital/az_quickstart
# Closes #1613: Add space to footer menu items

## Description
Add a space between footer list items

## Related issues
Closes #1613 

## How to test
https://b1f8212a-7d02-4d91-83c8-6d0013ee6896.probo.build/

1. Add menu items with long titles
2. Add menu to footer
3. Verify menu title wraps and has spacing between items

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
